v2/plugins/che-machine-exec-plugin/0.0.1/meta.yaml

```diff
@@ -0,0 +1,16 @@
+publisher: ""
+name: che-machine-exec-plugin
+version: 0.0.1
+type: Che Plugin
+displayName: Che machine-exec Service
+title: Che machine-exec Service Plugin
+description: Che Plug-in with che-machine-exec service to provide creation terminal
+  or tasks for Eclipse CHE workspace machines.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+url: https://github.com/eclipse/che-machine-exec/releases/download/0.0.2/che-service-plugin.tar.gz
+repository: https://github.com/eclipse/che-machine-exec/
+firstPublicationDate: "2019-02-05"
+category: Other
+deprecate:
+  autoMigrate: false
+  migrateTo: eclipse/che-machine-exec-plugin/0.0.1
```

Comment:
`migrateTo` is specified while `autoMigrate` is false. Could you please elaborate on how `migrateTo` is used in such case?

Reply:
It is not used ATM, but I suppose that UD should show warning that plugin is deprecated and another one should be used instead. The same warning can be produced by the broker.